rst2pdf/rson.py

```diff
@@ -151,8 +151,12 @@ def newstring(source, client):
                 source = source.encode('utf-8')
 
             # Convert MS-DOS or Mac line endings to the one true way
-            source = source.replace('\r\n', '\n').replace('\r', '\n')
-            sourcelist = splitter(source)
+            if six.PY2:
+                source = source.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
You can use b"bytes" in python2. At least in python 2.7 which honestly is the only one worth supporting nowadays.

Reply:
I don't intend to make any fancy changes to support Python2.X goal is to reach a state where the tests work on Python3. If we do this, we might be able to attract more people to actually contribute to the project, and people can use it in modern projects.